pkg/testutil/testutil_linux.go

```diff
@@ -57,7 +57,8 @@ var (
 )
 
 const (
-	FedoraESGZImage = "ghcr.io/stargz-containers/fedora:30-esgz" // eStargz
+	FedoraESGZImage = "ghcr.io/stargz-containers/fedora:30-esgz"                                                                             // eStargz
+	FfmpegSociImage = "public.ecr.aws/soci-workshop-examples/ffmpeg:sha256-4bd5d3b7143c5401f3b433bbd708d8a5f3eb72aef65a977a1e52e97c721b7687" // SOCI
```

Comment:
I guess this is SOCI index instead of the image. Maybe you want to pull the image ":latest".

Reply:
Ok, I'll try that